packages/blade/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import type { DividerProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import { useTheme } from '~components/BladeProvider';
+import type { ColorContrast } from '~tokens/theme/theme';
+import { makeSize } from '~utils';
+
+const Divider = ({
+  orientation = 'horizontal',
+  style = 'solid',
+  variant = 'normal',
+  thickness = 'thin',
+  contrast = 'low',
+}: DividerProps): React.ReactElement => {
+  const { theme } = useTheme();
+
+  const isDividerHorizontal = orientation === 'horizontal';
+  const borderPosition = isDividerHorizontal ? 'borderBottom' : 'borderLeft';
+
+  const borderWidth =
+    thickness === 'thinner' ? makeSize(0.5) : makeSize(theme.border.width[thickness]);
```

Comment:
Can you add a TODO "Tokenize thinner border width" and get this 0.5 from size[0.5], (add it to the size object if it's not there already)